llvm/utils/filecheck_lint/filecheck_lint.py

```diff
@@ -78,32 +78,41 @@ def levenshtein(s1: str, s2: str) -> int:  # pylint: disable=g-doc-args
 
 
 class FileRange:
-    """Stores the coordinates of a span on a single line within a file.
```

Comment:
We only "derive" in `as_str`, so I think this docstring should still say "Stores".